local_pathfinding/objectives.py

```diff
@@ -233,11 +235,15 @@ def heading_path_turn_cost(self, s1: ob.SE2StateSpace, s2: ob.SE2StateSpace):
 
 
 class WindObjective(Objective):
-    """Generates a wind objective function"""
+    """Generates a wind objective function
 
-    def __init__(self, space_information, wind_direction_degrees):
+    Attributes:
+        wind_direction (float): The direction of the wind in radians
```

Comment:
Range is [-pi, pi)

https://github.com/UBCSailbot/custom_interfaces/blob/main/msg/WindSensor.msg